src/validators/v1/actions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * name : validators/v1/actions.js
+ * author : Priyanka Pradeep
+ * Date : 14-Aug-2024
+ * Description : Validations of actions controller
+ */
+
+module.exports = {
+	update: (req) => {
+		req.checkParams('id')
+			.trim()
+			.optional()
+			.notEmpty()
+			.withMessage('id param is empty')
+			.isNumeric()
+			.withMessage('id param is invalid, must be an integer')
+
+		req.checkBody('code')
+			.notEmpty()
```

Comment:
Since we are passing id , can we make code optional?

Reply:
Whatever database field is mandatory to create record and it doesn't have default values, we can add that as a mandatory param in req body. So it wont throw error while creation or updation